test/src/test/java/hudson/model/UsageStatisticsTest.java

```diff
@@ -40,11 +44,16 @@
 /**
  * @author Kohsuke Kawaguchi
  */
-public class UsageStatisticsTest extends HudsonTestCase {
+public class UsageStatisticsTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
```

Comment:
I suspected something like that. Another common case is code which calls `Secret`, which requires `Jenkins.instance` for the master keystore.